File: platform/sdk/nodejs/examples/graph/graph-002_cdc_next_blocking.js
const { GraphClient } = require("../../xuanji-sdk-graph");
const client = new GraphClient({ endpoint: "graph.local" });
client.cdcNew("consumer-002");
const result = client.cdcNextBlocking("consumer-002");
if (!result.ok || result.events.length === 0) process.exit(1);
console.log("XJ-OK: graph-002_cdc_next_blocking");
process.exit(0);

File: platform/sdk/python/examples/graph/graph-002_cdc_next_blocking.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from mox_sdk_graph import GraphClient

client = GraphClient({"endpoint": "graph.local"})
client.cdcNew("consumer-002")
result = client.cdcNextBlocking("consumer-002")
if not result["ok"] or len(result["events"]) == 0:
    sys.exit(1)
print("XJ-OK: graph-002_cdc_next_blocking")
sys.exit(0)
